internal/pkg/mount/switchroot/switchroot.go

```diff
@@ -28,6 +30,9 @@ var preservedPaths = map[string]struct{}{
 	constants.SDStubDynamicInitrdPath: {},
 }
 
+//go:embed policy.33
```

Comment:
I think with this file we need to follow common `generate` approach - that is we should have it generated, put back to the source and committed to the repository.

Otherwise Talos won't be buildable with `go` tools outside of buildkit

Reply:
"selinux/policy.33"

Reply:
move to `internal/pkg/selinux`, embed as a variable, have a function which returns the value (policy)